timesync/manual_clock.go

```diff
@@ -0,0 +1,28 @@
+package timesync
+
+import "time"
+
+type ManualClock struct {
+	*Ticker
+	genesisTime time.Time
+}
+
+func (t *ManualClock) Tick() {
+	t.Notify()
```

Comment:
notify will not work since it's time edpendant. please run multi node sim

Reply:
Fixed